types/klaw-sync/klaw-sync-tests.ts

```diff
@@ -12,9 +12,20 @@ const defaultOptions = {}
 klawSync('/some/dir', defaultOptions).forEach(outputMessage)
 
 const options = {
-  ignore: ['.exe'],
-  nodir: true,
-  nofile: false,
+    ignore: ['.exe'],
+    nodir: true,
+    nofile: false,
 }
 
 klawSync('/some/dir', options).forEach(outputMessage)
+
+const optionsV2 = {
```

Comment:
I'd just refresh the old `options` hash above to reflect what's currently supported.